examples/pypi/run.sh

```diff
@@ -0,0 +1,22 @@
+set -x
+set -e
+
+# Start artipie.
+docker run --name artipie -d -it -v $(pwd)/artipie.yaml:/etc/artipie.yml -v $(pwd):/var/artipie -p 8080:80 artipie/artipie:latest
+
+# Wait for container to be ready for new connections.
+sleep 5
+
+# Build and upload python project to artipie.
+cd sample-project
+python3 -m pip install --user --upgrade setuptools wheel
+python3 setup.py sdist bdist_wheel
+python3 -m pip install --user --upgrade twine
```

Comment:
Can we use:

1. `pip`  instead of `python3 -m pip`?
2.  `twine` instead of `python3 -m twine`?
3.  pip instead of `python -m pip`?

Same about readme.

Reply:
I do not think we should get rid of `python3 -m` in the script as it explicitly shows python version we are working with here and might (if python3 is not default one) be required if someone will run this script locally. 
In readme, on the other hand, python version does not matter, so I agree we should remove it.